### DISCUSSION TITLE
How to disable commons?

### DISCUSSION THREAD

**Discussion by ilienkors** [vercel/next.js#40084] [Help]
Hello! I use Next.js as "landing selector" with single page. I have something like this

```
landingA: dynamic(() => import('landings/landingA')),
landingB: dynamic(() => import('landings/landingB')),
landingC: dynamic(() => import('landings/landingC')),
```
When someone goes to one landing, the code from other landings should not end up in commons because user won't see them. However user download big commons file. How can I deal with it?

Thanks for the help!

**Comment by icyJoseph** [comment]
Got a repo to look at? I think there might be shared code between these pages that gets all bundled, but I'd like to see a repo if possible.

**Reply by ilienkors** [reply]
@icyJoseph unfortunately the code is private.
Yes, there are common components between landings, but commons.js gets both the code of common components and other landings that are not related to each other.
